quarkus-opentelemetry-exporter-sentry/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.opentelemetry.exporter</groupId>
+        <artifactId>quarkus-opentelemetry-exporter-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-opentelemetry-exporter-sentry-parent</artifactId>
+    <packaging>pom</packaging>
+    <name>Quarkus Opentelemetry Exporter - Sentry</name>
+
+    <modules>
+        <module>runtime</module>
+        <module>deployment</module>
+        <module>integration-tests</module>
```

Comment:
Please remove the `integration-tests` and add activate it bellow in the profile.

This is because of the new release process, in order not to release IT tests.